duckduckgo/iOS
# Remove static disconnect list from project

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/375879998104269

**Description**:
• removes disconnect list from the project, now downlods it from the server instead
• adds error handling flow to cover rare situation where content blocker is opened before trackers list have ever been downloaded

**Steps to test this PR**:

Ensure that error page is shown if content blocker not yet retrieved:
1. Turn on aeroplane mode and freshly install app
2. Wait for it to launch
3. Turn network back on and search for something
4. Open content blocker, it should show an error page
5. Press the refresh button to retrieve the trackers list and show standard panel

Ensure content blocker continues to work as expected:
1. Launch app
2. Navigate to a page with trackers, ensure tracker number is updated

###### Reviewer Checklist:


###### PR DRI Checklist:
